<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Door Announcer Controller</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>390</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <layout class="QFormLayout" name="layout">
      <item row="1" column="0">
       <widget class="QLabel" name="portLabel">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="portBox">
        <property name="text">
         <string>9999</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="addressBox">
        <property name="text">
         <string>192.168.200.200</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="addressLabel">
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QComboBox" name="clipsBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>170</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="playClipButton">
        <property name="text">
         <string>Play</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="updateButton">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Door is:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="doorLabel">
        <property name="text">
         <string>Unknown</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="toggleDoorButton">
        <property name="text">
         <string>Toggle Door State</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openDoorButton">
        <property name="text">
         <string>Set Door to Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeDoorButton">
        <property name="text">
         <string>Set Door to Closed</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>addressBox</tabstop>
  <tabstop>portBox</tabstop>
  <tabstop>updateButton</tabstop>
  <tabstop>clipsBox</tabstop>
  <tabstop>playClipButton</tabstop>
  <tabstop>toggleDoorButton</tabstop>
  <tabstop>openDoorButton</tabstop>
  <tabstop>closeDoorButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
